docs/integrations.md

```diff
@@ -156,3 +156,25 @@ In addition, ruff publishes the following images:
 
 As with the distroless image, each image is published with ruff version tags as
 `ruff:{major}.{minor}.{patch}-{base}` and `ruff:{major}.{minor}-{base}`, e.g., `ruff:0.6.6-alpine`.
+
+## GitLab CI/CD
+
+You can add the following configuration to `.gitlab-ci.yml` to run a `ruff format`, and a `ruff check` compatible with GitLab's codequality.
+
+```yaml
+Ruff:
+  stage: .pre
+  interruptible: true
+  image:
+    name: ghcr.io/astral-sh/ruff:0.7.1-alpine
```

Comment:
What's the motivation of using the docker image over using e.g. pip or uv pip to install the project's ruff version? 
That would be more aligned with our GitHub Actions example

Reply:
It's _way_ faster than installing ruff with `pip` every single time. That's one of the major advantages of the new images with extra tags you guys push.

Reply:
That makes sense. It does have the downside that they have to update the ruff version number locally in the project dependencies and CI. But I guess we can always add a second example if needed.